internal/services/datafactory/data_factory_dataset_snowflake_resource.go

```diff
@@ -107,7 +107,7 @@ func resourceDataFactoryDatasetSnowflake() *pluginsdk.Resource {
 				},
 			},
 
-			"schema_column": {
+			"structure_column": {
```

Comment:
we should probably deprecate this property?

Reply:
Thanks @katbyte, I added the deprecated warning.